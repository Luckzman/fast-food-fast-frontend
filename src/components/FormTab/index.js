import React from 'react';
import { Tab } from 'semantic-ui-react';
import LoginForm from '../LoginForm';
import SignupForm from '../SignupForm';

const panes = [
  {
    menuItem: 'Signup',
    render: function login() {
      return (
        <Tab.Pane attached={false}>
          <SignupForm />
        </Tab.Pane>
      );
    }
  },
  {
    menuItem: 'Login',
    render: function signup() {
      return (
        <Tab.Pane attached={false}>
          <LoginForm />
        </Tab.Pane>
      );
    }
  }
];

const FormTab = () => <Tab menu={{ secondary: true }} panes={panes} />;

export default FormTab;
