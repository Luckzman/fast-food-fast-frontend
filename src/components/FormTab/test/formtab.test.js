import React from 'react';
import { shallow } from 'enzyme';
import FormTab from '..';

describe('App', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<FormTab />);
    expect(wrapper).toMatchSnapshot();
  });
});
