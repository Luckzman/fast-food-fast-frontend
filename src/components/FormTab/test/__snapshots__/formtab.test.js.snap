// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<Tab
  grid={
    Object {
      "paneWidth": 12,
      "tabWidth": 4,
    }
  }
  menu={
    Object {
      "secondary": true,
    }
  }
  panes={
    Array [
      Object {
        "menuItem": "Signup",
        "render": [Function],
      },
      Object {
        "menuItem": "Login",
        "render": [Function],
      },
    ]
  }
  renderActiveOnly={true}
/>
`;
